cpp/open3d/visualization/utility/SelectionPolygonVolume.h

```diff
@@ -64,13 +64,14 @@ class SelectionPolygonVolume : public utility::IJsonConvertible {
     std::shared_ptr<geometry::TriangleMesh> CropTriangleMesh(
             const geometry::TriangleMesh &input) const;
 
+    std::vector<size_t> CropInPolygon(
+            const std::vector<Eigen::Vector3d> &input) const;
```

Comment:
If this method is set public, I think it would be better to change input from `std::vector<Eigen::Vector3d>` to `geometry::PointCloud`.

Reply:
Sounds good, added a public api to take `geometry::PointCloud` as input, which internally uses the private api to crop.